scripts/test-build-zip.sh

```diff
@@ -46,6 +77,10 @@ done
 # Remove unzipped built packages, leave .zip files
 rm -r build/packages/*/
 
+if [[ "${SKIP_INSTALL}" == 1 ]]; then
+    exit 0
+fi
```

Comment:
Nit. Instead of having this if and the ones in `with_docker` and `with_docker_compose`, maybe we can add a new script that just builds the zip and doesn't call to `with_docker` and `with_docker_compose`. Even if we have some duplication between the scripts, it would be more explicit. WDYT?

Reply:
Sure, I'll do that. It is going to be better to have that script with less "ifs"